src/pages/Home.js

```diff
@@ -215,8 +216,11 @@ class Home extends Component {
                       a({
                         href: 'https://www.ga4gh.org/genomic-data-toolkit/regulatory-ethics-toolkit/#:~:text=Machine%20Readable%20Consent%20Guidance&text=Machine%20readable%20consent%20language%20is,to%20for%20their%20research%20purposes',
                         target: '_blank'
-                      }, ['Machine Readable Consent Guidance.'])
-                    ]),
+                      }, ['Machine Readable Consent Guidance.']),
+                      " For help determining your data's permitted uses, try our ",
+                      (Link, { to: '/data_sharing_language_tool', className: 'hover-color'}, ['Data Sharing Language Tool']),
```

Comment:
Link needs to be wrapped to use this construction syntax (see https://www.npmjs.com/package/react-hyperscript-helpers) 
```
h(Link, { to: '/data_sharing_language_tool', className: 'hover-color'}, ['Data Sharing Language Tool'])
```